core/src/backends/eclair/rest/config.rs

```diff
@@ -1,4 +1,4 @@
-use crate::types::NodeConfig;
+use crate::{error::ConfigError, types::NodeConfig};
```

Comment:
```suggestion
use crate::error::ConfigError;
use crate::types::NodeConfig;
```
I kinda like it like this, separating imports by crate. Not sure if it is better.